src/lib/cli/command.js

```diff
@@ -22,6 +22,13 @@ import { trackEvent } from 'lib/tracker';
 import pager from 'lib/cli/pager';
 import { parseEnvAliasFromArgv } from './envAlias';
 
+const Rollbar = require( 'rollbar' );
+const rollbar = new Rollbar( {
+	accessToken: '0d8145a4b91d4e0886bad98781dddb64',
+	captureUncaught: true,
+	captureUnhandledRejections: true,
```

Comment:
Can we differentiate dev vs "production" (i.e. installed from npm)? Maybe similar to how we handle Tracks?

Reply:
oups forgot about this. Yes, there is an `environment` flag in the constructor https://docs.rollbar.com/docs/nodejs#section-standalone

I think the env part need to be part of the config we ship with the package in `config/`

Reply:
Another thought: it would be helpful to have the CLI version number as well (maybe in the `payload` object https://docs.rollbar.com/docs/rollbarjs-configuration-reference#section-payload ?)